siliconcompiler/project.py

```
@@ -586,9 +586,23 @@ def run(self) -> TProject:
+                # output), and unregisters the atexit hook, so this project is no
+                # longer pinned and can be garbage collected. update_manifest()
+                # does a schema retraversal that can raise (I/O / serialization);
+                # a failing final repaint must not skip teardown, nor mask the
+                # run's own result or error, so it is guarded. The shared Board
+                # is a process-wide singleton: its completeness guard keeps it
+                # alive for other concurrent runs and only truly stops it once
+                # all jobs are done; MPManager.stop() is the process-exit backstop.
+                try:
+                    self.__dashboard.update_manifest()
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)